pkg/helpers/tokencmd/request_token.go

```diff
@@ -225,13 +225,23 @@ func (o *RequestTokenOptions) RequestToken() (string, error) {
 		defer resp.Body.Close()
 
 		if resp.StatusCode == http.StatusUnauthorized {
-			if resp.Header.Get("WWW-Authenticate") != "" {
+			if len(resp.Header.Get("WWW-Authenticate")) > 0 {
 				if !o.Handler.CanHandle(resp.Header) {
 					return "", apierrs.NewUnauthorized("unhandled challenge")
 				}
 				// Handle the challenge
 				newRequestHeaders, shouldRetry, err := o.Handler.HandleChallenge(requestURL, resp.Header)
 				if err != nil {
+					if IsBasicNoUsernameError(err) {
+						tokenPromptErr := apierrs.NewUnauthorized("")
+						klog.V(4).Infof("%v", err)
+						tokenPromptErr.ErrStatus.Details = &metav1.StatusDetails{
+							Causes: []metav1.StatusCause{
+								{Message: fmt.Sprintf("You must obtain an API token by visting %s/request\n", o.OsinConfig.TokenUrl)},
```

Comment:
Remove trailing newline, fix to `visiting`.  Is this consistent with the server message?

Reply:
yes, but i'll double check

Reply:
oops fixed spelling
removed newline, but checking w/ the 3.11 api.ci cluster server msg has a newline

Reply:
https://github.com/openshift/oauth-server/blob/master/pkg/authenticator/challenger/placeholderchallenger/placeholder_challenger.go#L23-#L30